chainsail_helpers/chainsail_helpers/pdf/__init__.py

```diff
@@ -34,4 +37,94 @@ def log_prob_gradient(self, x):
             np.ndarray: 1D array of floats containing the flattened
               log-probability gradient evaluated at x
         """
+        passs
```

Comment:
```suggestion
        pass
```

Reply:
I am starting to get embarrassed :sweat_smile: thanks!